cleanlab/internal/util.py

```diff
@@ -681,12 +726,22 @@ def num_unique_classes(labels, multi_label=None) -> int:
     the format of labels.
     This allows for a more general form of multiclass labels that looks
     like this: [1, [1,2], [0], [0, 1], 2, 1]"""
+    return len(get_unique_classes(labels, multi_label))
+
+
+def get_unique_classes(labels, multi_label=None) -> set:
+    """Returns the set of unique classes for both single-labeled
+    and multi-labeled labels. If multi_label is set to None (default)
+    this method will infer if multi_label is True or False based on
+    the format of labels.
+    This allows for a more general form of multiclass labels that looks
```

Comment:
should we extend this to all multilabel formats?
just a question, I can do this once it's merged

Reply:
only if you need it for multilabel find-label-issues to work, when some of the classes never appear in data.

Our main priority is just to get multilabel find-label-issues to work when some of the classes never appear in data.